lib/logger.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+var Log;
+
+Log = require("log");
+
+module.exports = (args) => {
```

Comment:
I like having the library logging available to the user, just in case. So I think it's okay to leave this little wrapper in. Though, I'm open to being convinced otherwise.